src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -44,7 +44,7 @@
 
 vsts_account_name = CliArgumentType(
     options_list=('--vsts-account-name',),
-    help='VSTS account name to create the build and release definitions. A new VSTS account is created if omitted or does not exist.'
+    help='VSTS account name to create the build and release definitions. If you have an existing VSTS account at myvstsaccount.visualstudio.com, then set this parameter value to myvstsaccount. A new VSTS account is created if parameter value is omitted or does not exist.'
 )
 
 vsts_project_name = CliArgumentType(
```

Comment:
Since that command doesn't use those parameters, it will not cause an issue. Aliases are ignored if there is no matching parameter.

If it were an issue or you just wanted to be more precise, you could also use this approach:
```Python
for item in ['build', 'release']:
  register_cli_argument('container {} create'.format(item), 'target_name', target_name)
```
Then you would still have only a single set of these definitions. In this case, where the help text is very specific, this more precise approach might be better, but for what you have, either is fine.